hpy() fails when "coverage" is loaded

hpy() raises an AttributeError when coverage is running code that uses hpy, because the CoverageDebugFile module has no __dict__ attribute:

```
.tox/py37/lib/python3.7/site-packages/guppy/__init__.py:36: in hpy
    return r.guppy.heapy.Use
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:50: in __getattr__
    return self._share.getattr(self, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:218: in getattr
    d = self.getattr2(inter, cache, owner, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:235: in getattr2
    x = self.getattr_package(inter, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:283: in getattr_package
    x = self.makeModule(x, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:349: in makeModule
    return Share(module, self, module.__name__, Clamp)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:207: in __init__
    getattr(inter, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:50: in __getattr__
    return self._share.getattr(self, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:218: in getattr
    d = self.getattr2(inter, cache, owner, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:237: in getattr2
    x = self.getattr3(inter, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:322: in getattr3
    pa = getattr(pa, at)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:50: in __getattr__
    return self._share.getattr(self, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:218: in getattr
    d = self.getattr2(inter, cache, owner, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:235: in getattr2
    x = self.getattr_package(inter, name)
.tox/py37/lib/python3.7/site-packages/guppy/etc/Glue.py:272: in getattr_package
    x = __import__(self.makeName(name))
.tox/py37/lib/python3.7/site-packages/guppy/heapy/View.py:571: in <module>
    prime_builtin_types()
.tox/py37/lib/python3.7/site-packages/guppy/heapy/View.py:551: in prime_builtin_types
    for t in list(mod.__dict__.values()):
E   AttributeError: 'tuple' object has no attribute '__dict__'
```

The offending module is a tuple:

```
(<coverage.debug.DebugOutputFile object at 0x7fec9fb98668>, False)
```

which has no __dict__ attribute.

Would you accept a PR to look for the __dict__ attribute in View.py and skip anything that doesn't have that attribute?

Thanks

--- a/guppy/heapy/View.py
+++ b/guppy/heapy/View.py
@@ -539,7 +539,7 @@ def prime_builtin_types():
     import weakref
 
     for mod in list(sys.modules.values()):
-        if mod is None:
+        if mod is None or getattr(mod, '__dict__', None) is None:
             continue
         for t in list(mod.__dict__.values()):
             if isinstance(t, type):

